types/k6/experimental/browser.d.ts

```diff
@@ -0,0 +1,1508 @@
+
+    /**
+     * When `true`, the call requires selector to resolve to a single element.
+     * If given selector resolves to more than one element, the call throws
+     * an exception. Defaults to `false`.
+     */
+    strict?: boolean;
+
+    /**
+     * Maximum time in milliseconds. Defaults to `0` - no timeout. Default is
+     * overridden by the `setDefaultTimeout` option on `BrowserContext` or
+     * `page` methods.
+     */
+    timeout?: number;
+  }): void;
+
+  /**
+   * **NOTE** Use locator-based `locator.focus([options])` instead.
+   *
+   * This method fetches an element with `selector` and focuses it.
+   *
+   * @param selector A selector to search for an element. If there are multiple
+   * elements satisfying the selector, the first will be used.
+   * @param options
+   */
+  focus(selector: string, options?: {
+    /**
+     * When `true`, the call requires selector to resolve to a single element.
+     * If given selector resolves to more than one element, the call throws
+     * an exception. Defaults to `false`.
+     */
+    strict?: boolean;
+
+    /**
+     * Maximum time in milliseconds. Defaults to `0` - no timeout. Default is
+     * overridden by the `setDefaultTimeout` option on `BrowserContext` or
+     * `page` methods.
+     */
+    timeout?: number;
+  }): void;
+
+  /**
+   * Frames returns an array of frames on the page.
+   */
+  frames(): Frame[];
+
+  /**
+   * **NOTE** Use locator-based locator.getAttribute(name[, options]) instead.
+   *
+   * Returns the element attribute value for the given attribute name.
+   *
+   * @param selector A selector to search for an element. If there are multiple
+   * elements satisfying the selector, the first will be used.
+   * @param name Attribute name to get the value for.
+   * @param options
+   */
+  getAttribute(selector: string, name: string, options?: {
+    /**
+     * When `true`, the call requires selector to resolve to a single element.
+     * If given selector resolves to more than one element, the call throws
+     * an exception. Defaults to `false`.
+     */
+    strict?: boolean;
+
+    /**
+     * Maximum time in milliseconds. Defaults to `0` - no timeout. Default is
+     * overridden by the `setDefaultTimeout` option on `BrowserContext` or
+     * `page` methods.
+     */
+    timeout?: number;
+  }): null|string;
+
+  /**
+   * Navigates to the specified url and returns the main resource response.
+   *
+   * navigating to `about:blank` or navigation to the same URL with a different
+   * hash, will succeed and return `null`.
+   *
+   * @param url URL to navigate page to. The url should include scheme, e.g.
+   * `https://`.
+   * @param options
+   */
+  goto(url: string, options?: {
+    /**
+     * Referer header value.
+     */
+    referer?: string;
+
+    /**
+     * Maximum operation time in milliseconds. Defaults to `0` - no timeout. The
+     * default value can be changed via `navigationTimeout` option in the config,
```

Comment:
I believe this `navigationTimeout` option in the config does not apply to our implementation right?